client/src/components/User.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import UserExtension from "./UserExtension.jsx";
+
+export default class User extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div>
+        <div>
```

Comment:
Try and avoid duplicate, unnamed <div> elements. It's good to either provide a css selector or simplify your structure. It's good practice to provide a css selector to at least your component's most parent element. That way you can easily manipulate the entire component at will.

Reply:
@kimbyongjin duly noted ty